paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/PaimonBaseScan.scala

```diff
@@ -72,4 +95,23 @@ abstract class PaimonBaseScan(table: Table, readBuilder: ReadBuilder, desc: Stri
     }
     super.supportedCustomMetrics() ++ paimonMetrics
   }
+
+  override def description(): String = {
+    val requiredFieldNameStr = readSchema().map(_.name).mkString(",")
+    val pushedFiltersStr = if (filters.nonEmpty) {
+      ", PushedFilters: " + filters.map(_._1).mkString(",")
+    } else {
+      ""
+    }
+    val runtimeFiltersStr = if (runtimeFilters.nonEmpty) {
+      ", RuntimeFilters: " + runtimeFilters.mkString(",")
+    } else {
+      ""
+    }
+    super.description() + "["
+    "Fields: " + requiredFieldNameStr + pushedFiltersStr +
+      runtimeFiltersStr +
```

Comment:
maybe will we don't need requiredFieldNameStr and runtimeFiltersStr

Reply:
They are not always same in case querying nested column or unsupported runtime filter.